app/src/main/java/com/example/moviewithfragments/models/MovieModel.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.moviewithfragments.models
+
+import android.os.Parcelable
+import com.google.gson.annotations.SerializedName
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+class MovieModel(
+    @SerializedName("id")
+    val movieId: Int,
+    @SerializedName("original_title")
+    val title: String,
+    @SerializedName("overview")
+    val overview: String,
+    @SerializedName("poster_path")
+    val posterPath: String,
+    @SerializedName("release_date")
+    val releaseDate: String?,
+    @SerializedName("vote_average")
+    val voteAverage: Float
+) : Parcelable {
```

Comment:
We should not use same data classes at network layer and UI layer.
for example releaseDate in network layer is string
but in in UI year it can be of type Date.